src/scheduler/annualReviewStartEmailScheduler.ts

```diff
@@ -0,0 +1,211 @@
+import { findListByAnnualReviewDate } from "server/models/list";
+import {
+  List, ListItemGetObject,
+} from "server/models/types";
+import { logger } from "server/services/logger";
+import { findListItemsForLists, updateAnnualReview } from "server/models/listItem";
+import {
+  sendAnnualReviewPostEmail, sendAnnualReviewProviderEmail,
+} from "server/services/govuk-notify";
+import { BaseDeserialisedWebhookData } from "server/models/listItem/providers/deserialisers/types";
+import { initialiseFormRunnerSession } from "server/components/formRunner/helpers";
```

Comment:
The flow is now 
1. Provider gets email so they can confirm their changes
2. Provider firsts lands on `lists` app, asking if they are still qualified
3. Then they get a summary page of their details on the `lists` app
4. They answer if their details have changed
5. If they have changed, then a token is generated and they are redirected to the `runner`.


I think you will need to consolidate with @RichardBray's code. As long as you're not emailing a link that sends a user straight to the runner, it will be ok. I seem to remember @RichardBray generating a token via the runner, just not sending them straight there.

Reply:
ah that's right.  Rather than invoke merge hell, probably worth just setting the path to Rich's landing page here instead and pull in when his changes are ready